enforcer-rules/src/main/java/org/apache/maven/plugins/enforcer/BannedDependenciesBase.java

```diff
@@ -20,12 +20,12 @@
 
 import java.util.List;
 
-import org.apache.commons.lang3.StringUtils;
 import org.apache.maven.artifact.Artifact;
 import org.apache.maven.enforcer.rule.api.EnforcerRuleException;
 import org.apache.maven.enforcer.rule.api.EnforcerRuleHelper;
 import org.apache.maven.execution.MavenSession;
 import org.apache.maven.plugins.enforcer.utils.ArtifactUtils;
+import org.apache.maven.shared.utils.StringUtils;
```

Comment:
Why change commons-lang to Maven shared?
Even more I would to like remove Maven shared from dependencies in the future.

Reply:
Even worse. The reason was my IDE changing it to Google something during a failed rebase, which was the reason for the red build. Corrected.